src/api/RequestApi.ts

```diff
@@ -255,7 +255,8 @@ export type IInRequest = {
   supGovLead: SPPersona;
   /** Required - The employee GAL entry. If the user doesn't exist yet, then it will be undefined */
   employee: SPPersona | undefined;
-  isTraveler: "yes" | "no";
+  /** Required - Can only be 'true' if it is a Civ/Mil. Must be 'false' if it is not a 'civ' or 'mil' */
```

Comment:
Update comment to
/** Required - Can only be 'true'/'false' if it is Civilian or Military.  Must be '' if it is a Contractor */